docs/src/development/variables/_index.md

```diff
@@ -0,0 +1,198 @@
+All variables can also be [overridden via script](./set-variables.md#set-variables-via-script).
+
+For an example of how the different levels work,
+suppose you have the following inheritable variables defined for the `main` environment:
+
+```sh
+$ platform var -e main
+Variables on the project Example (abcdef123456), environment main:
++----------------+-------------+--------+---------+
+| Name           | Level       | Value  | Enabled |
++----------------+-------------+--------+---------+
+| system_name    | project     | Spiffy |         |
+| system_version | project     | 1.5    |         |
+| api_key        | environment | abc123 | true    |
+| debug_mode     | environment | 1      | true    |
++----------------+-------------+--------+---------+
+```
+
+And the following variables defined for the `feature-x` environment, a child environment of `main`:
+
+```sh
+$ platform var -e feature-x
+Variables on the project Example (abcdef123456), environment feature-x:
++----------------+-------------+--------+---------+
+| Name           | Level       | Value  | Enabled |
++----------------+-------------+--------+---------+
+| system_name    | project     | Spiffy |         |
+| system_version | project     | 1.5    |         |
+| api_key        | environment | def456 | true    |
+| system_version | environment | 1.7    | true    |
++----------------+-------------+--------+---------+
+```
+
+In the `main` environment, you can access `$PLATFORM_VARIABLES`:
+
+```bash
+echo $PLATFORM_VARIABLES | base64 --decode | jq
+```
+
+The output looks like this:
+
+```json
+{
+    "system_name": "Spiffy",
+    "system_version": "1.5",
+    "api_key": "abc123",
+    "debug_mode": "1"
+}
+```
+
+While in the `feature-x` environment, it looks like this:
+
+```json
+{
+    "system_name": "Spiffy",
+    "system_version": "1.7",
+    "api_key": "def456",
+    "debug_mode": "1"
+}
+```
+
+To get a visual overview of which variables are overridden in an environment,
+navigate in the management console to that environment's variables settings.
+This example shows how it looks within the `feature-x` environment:
+
+<!-- vale Vale.Spelling = NO -->
+<!-- spelling turned off because of the "api_key" -->
+![The console showing the variables split into environment and project ones, with the environment variables `api_key` and `system_version` labeled as overridden and `debug_mode` as inherited the project variable `system_version` labeled as inactive.](/images/management-console/variables-overridden.png "0.5")
+<!-- vale Vale.Spelling = YES -->
+
+Project variables that conflict with environment variables are labeled as **Inactive**.
+Environment variables are labeled as **Inherited** when they get their value from a parent environment
+and as **Overridden** when there is a conflict and the parent environment's value doesn't apply.
+
+## Variable prefixes
+
+Certain variable name prefixes have special meanings.
+Some are defined by Platform.sh and apply automatically.
```

Comment:
```suggestion
Some are defined by Platform.sh and apply automatically ( such as `PLATFORM_<name of variable>`).
```

Reply:
Those ones are treated as a separate case and aren't what's meant here. It's prefixes like `env` here that apply. If you add `PLATFORM_` as a prefix, I don't think it does anything special.

Reply:
ah, indeed that makes more sense!